student.rb

```diff
@@ -1,11 +1,18 @@
 require_relative 'Person'
 
+attr_accessor :classroom
+
```

Comment:
- [ ] We just have to set the getter for the `classroom` instance variable. Please update it.

Hint:
`attr_reader :classroom`